### DISCUSSION TITLE
ENOENT: no such file or directory, open

### DISCUSSION THREAD

**Discussion by pgm-norbert** [FlowiseAI/Flowise#637] [Q&A]
I get the following errors when trying to run the chatbot - I have enabled full access to the node_modules directory but still getting the same error. Any ideas? 

Error: Error: ENOENT: no such file or directory, open ''
Error: Error: Error: ENOENT: no such file or directory, open ''
    at getCredentialData (/Users/userx/node_modules/flowise-components/dist/src/utils.js:448:15)
    at async OpenAIEmbedding_Embeddings.init (/Users/userx/node_modules/flowise-components/dist/nodes/embeddings/OpenAIEmbedding/OpenAIEmbedding.js:57:32)
    at async buildLangchain (/Users/userx/node_modules/flowise/dist/utils/index.js:220:50)
    at async App.processPrediction (/Users/userx/node_modules/flowise/dist/index.js:841:44)
    at async /Users/userx/node_modules/flowise/dist/index.js:557:13
2023-07-28 19:54:42 [ERROR]: [server]: Error: Error: Error: Error: ENOENT: no such file or directory, open ''
Error: Error: Error: Error: ENOENT: no such file or directory, open ''
    at buildLangchain (/Users/userx/node_modules/flowise/dist/utils/index.js:230:19)
    at async App.processPrediction (/Users/userx/node_modules/flowise/dist/index.js:841:44)
    at async /Users/userx/node_modules/flowise/dist/index.js:557:13
2023-07-28 19:54:42 [INFO]: ⬆️ POST /api/v1/chatmessage/882b7358-6903-4260-aa48-2ae32826e6d0

**Comment by HenryHengZJ** [comment]
issue should be resolved now. simply pull the latest docker image or reinstall flowise if using npx

**Reply by danclariteeai** [reply]
![Snip20230803_3](https://github.com/FlowiseAI/Flowise/assets/141318879/aae150a1-660b-4bc3-ab0c-1da6091fe0fc)

i'm getting the same message when i run the chatbot - Error: Error: Error: ENOENT: no such file or directory, open .

**Reply by GDeremaux** [reply]
Did you manage to fix it ?

**Reply by Canabiense** [reply]
I see the same error when trying to install via Command Prompt (npm install -g flowise and npm flowise start). It seems now all folders are loading/downloading properly.

works fine for me on Render.

**Reply by codar43** [reply]
I've been struggling to solve this issue too! Can you explain why it is and how to solve it step by step (if possible) cause I'm not a programmer unfortunately! 
![Flowise Error 2](https://github.com/FlowiseAI/Flowise/assets/141015641/56f57742-01f0-4c61-9e1a-a9b60692261a)

**Reply by codar43** [reply]
I'm not a programmer, can you elaborate more on this , more like a step by step guide to actually fix this! It would be great help Thank You @HenryHengZJ

**Comment by codar43** [comment]
> I see the same error when trying to install via Command Prompt (npm install -g flowise and npm flowise start). It seems now all folders are loading/downloading properly.
> 
> works fine for me on Render.

@Canabiense  "Hey ! Did Render ask for Card information from you, even if you use the individual (free) plan? Cause I'm trying to Access flowise using render and when I go to the step 2 which is to Create website service, Its ask for personal info! Can You help me with that?

**Reply by Canabiense** [reply]
no, it's free. there is an 'instance' called FREE that i can choose.

**Reply by codar43** [reply]
Yes! I did choose the Free plan but still not working! I think its an update policy of render!

**Comment by Canabiense** [comment]
![render_Capture](https://github.com/FlowiseAI/Flowise/assets/141098579/8a1e4c61-878f-41a9-b65b-827d170203c1)
